google/cloud/security/inventory/api_map.py

```diff
@@ -57,6 +57,12 @@
             'class_name': 'StorageClient',
             'version': None
         },
+    'gke_api':
```

Comment:
Doesn't 'container' need to be added to configs/forseti_conf.yaml.* too?

Reply:
Good catch, fixed in the conf yaml files.